SAP2000_Adapter/CRUD/Create/Loads.cs

```diff
@@ -591,7 +591,48 @@ private bool CreateLoad(BarPrestressLoad bhLoad)
 
             return true;
         }
-        
+
+        /***************************************************/
+
+        private bool CreateLoad(BarDifferentialTemperatureLoad bhLoad)
+        {
+            List<Bar> bars = bhLoad.Objects.Elements.ToList();
+            string loadPat = GetAdapterId<string>(bhLoad.Loadcase);
+            string patternName = "None";
+            bool replace = SAPPushConfig.ReplaceLoads;
+            int myType = 1;
+
+            double tempUniform = bhLoad.GetUniformComponent();
```

Comment:
Does SAP2000 also apply the temperature gradient at the midpoint of the Bar and you have to offset it with a uniform temperature?

Reply:
SAP2000 only allows constant gradients, so it has to start at 0, whereas the differential load in BHoM allows any profile.